<launch>
    <param name = 'device' value = '' />
    <param name = 'muscle_count' value = '1' />
    <param name = 'sampling_frequency' value = '512' />
    <param name = 'channels_to_remove' value = '' />
    <param name = 'file_dir' value = '$(find mobile_hdemg_exo)' />
    <param name = 'startup_gui_completed' value = 'False'/>
    <param name = 'connected_to_emg' value = 'False'/>
    <param name = 'emg_coef' value = '0.5'/>
    <node name = 'emg_stream_node' pkg = 'mobile_hdemg_exo' type = 'emg_stream_node.py' output = 'screen' />
    <node name = 'emg_processor_node' pkg = 'mobile_hdemg_exo' type = 'emg_processor_node.py' output = 'screen' />
    <node name = 'emg_visualizer_node' pkg = 'mobile_hdemg_exo' type = 'emg_visualizer_node.py' output = 'screen' />
    <node name = 'startup_gui_node' pkg = 'mobile_hdemg_exo' type = 'startup_gui_node.py' output = 'screen' />
    <!-- Optional -->
    <!-- <node name = 'torque_file_node' pkg = 'mobile_hdemg_exo' type = 'torque_file_node.py' output = 'screen' /> -->
</launch>
